src/Feed/Feed.scss

```diff
@@ -0,0 +1,8 @@
+.header-title {
+    font-family: 'Damion', cursive;
+    text-align: center;
+    font-size: 1.5em;
+    margin-top: 1em;
+    margin-bottom: 0em;
+    color: #026C92;
```

Comment:
Should we look at storing the colors in a central location and call the colors into the file?